article/app/views/fragments/recipeArticleBody.scala.html

```diff
@@ -0,0 +1,174 @@
+@import layout.ContentWidths.MainMedia
+@import model.content.MediaAtom
+@import common.Localisation
+@(articleModel: ArticlePage, recipes: Seq[model.content.RecipeAtom])(implicit request: RequestHeader, context: _root_.model.ApplicationContext)
+
+@import common.LinkTo
+@import views.BodyCleaner
+
+@schemaType(page: ArticlePage) = @{ page.article.metadata.schemaType }
+@bodyType(page: ArticlePage) = @{ "articleBody" }
+
+@defining(articleModel.article) { article =>
+    <article id="article" data-test-id="article-root"
+            class="content content--recipes"
+            itemscope itemtype="@schemaType(articleModel)" role="main" >
+    <meta itemprop="mainEntityOfPage" content="@LinkTo(article.metadata.url)">
```

Comment:
Nit: indentation should probably be changed so that meta is indented within article